Jenkinsfile

```diff
@@ -21,6 +21,7 @@ def releaseTag = new com.socrata.ReleaseTag(steps, service_server)
 pipeline {
   options {
     ansiColor('xterm')
+    disableConcurrentBuilds(abortPrevious: true)
```

Comment:
Bonus: helpful setting that we now use on other services.  If a new build starts while an existing build is in progress, the first build is cancelled.

Reply:
Is the premature termination always safe?

Reply:
Builds have failed in many places and it's never been unsafe or required additional cleanup.